<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图片拼接器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="image_list_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_img_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_img_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_arc_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>更改参数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="channce_changebutton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>410</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_change_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stitch_img_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>拼接图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_img_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_imgs_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清空图片</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="log_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>571</width>
      <height>191</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="wavecorrect_checkbox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WaveCorrect</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showimg_checkbox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ShowImage</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="saveimage_checkbox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SaveImage</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="preview_checkbox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="workmegapix_spinbox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WorkMegapix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SeamMegapix</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="seammegapix_spinbox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ComposeMegapix</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="composemegapix_spinbox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ConfThresh</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="confthresh_spinbox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MatchConf</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="matchconf_spinbox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="featurestype_combobox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FeaturesType</string>
    </property>
   </widget>
   <widget class="QComboBox" name="matchertype_combobox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MatcherType</string>
    </property>
   </widget>
   <widget class="QComboBox" name="bacosttype_combobox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="wavecorrecttype_combobox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="warptype_combobox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BaCostFuncType</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WaveCorrectType</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WarpType</string>
    </property>
   </widget>
   <widget class="QComboBox" name="exposcomptype_combobox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ExposCompType</string>
    </property>
   </widget>
   <widget class="QComboBox" name="seamfindtype_combobox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SeamFindType</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BlendStrength</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="blendstrength_spinbox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BlendType</string>
    </property>
   </widget>
   <widget class="QComboBox" name="blendtype_combobox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="showlog_checkbox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ShowLog</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
